plugin.py

```diff
@@ -62,15 +65,18 @@ def __init__(self, session: "weakref.ref[Session]") -> None:
         if sess:
             self.plugin_mapping[sess.window.id()] = self
 
-        # ST persists view setting after getting closed so we have to reset some status
-        for window in sublime.windows():
-            for view in window.views(include_transient=True):
-                erase_copilot_view_setting(view, "is_visible")
-                erase_copilot_view_setting(view, "is_waiting")
+        # ST persists view settings after getting closed so we have to reset some status
+        for view in all_views():
+            cm = ViewCompletionManager(view)
+            cm.is_visible = False
+            cm.is_waiting = False
```

Comment:
Maybe we should introduce the `reset` method: `ViewCompletionManager(view).reset()` (for the `ViewPanelCompletionManager` too)

Reply:
Done in [a168798](https://github.com/TheSecEng/LSP-copilot/pull/18/commits/a16879844cf092d263701cb321475b3e143e8044)